doc/cephfs/health-messages.rst

```diff
@@ -240,3 +240,15 @@ other daemons, please see :ref:`health-checks`.
   Description
     All MDS ranks are unavailable resulting in the file system to be completely
     offline.
+
+``MDS_CLIENTS_LAGGY``
+----------------------------
+  Message
+    "Client *ID* is laggy; not evicted because some OSD(s) is/are laggy"
+
+  Description
+    OSD(s) can be laggy at times and this can lead to laggy clients(session might
```

Comment:
Hmmm.. not at times but due to certain conditions (network cut-off, etc..) - better to specify that IMO.